JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -175,16 +180,36 @@ fun InterestsScreen(
     Scaffold(
         scaffoldState = scaffoldState,
         topBar = {
+
             InsetAwareTopAppBar(
-                title = { Text("Interests") },
+                title = {
+                    Text(
+                        text = stringResource(R.string.cd_interests),
+                        modifier = Modifier.fillMaxWidth(),
+                        textAlign = TextAlign.Center
+                    )
+                },
                 navigationIcon = {
                     IconButton(onClick = openDrawer) {
-                        Icon(
+                        Image(
                             painter = painterResource(R.drawable.ic_jetnews_logo),
-                            contentDescription = stringResource(R.string.cd_open_navigation_drawer)
+                            contentDescription = stringResource(R.string.cd_open_navigation_drawer),
+                            colorFilter = ColorFilter.tint(MaterialTheme.colors.primary)
                         )
                     }
-                }
+                },
+                actions = {
+                    IconButton(
+                        onClick = { /* TODO: Open search */ }
+                    ) {
+                        Icon(
+                            imageVector = Icons.Filled.Search,
+                            contentDescription = stringResource(R.string.cd_search)
+                        )
+                    }
+                },
+                backgroundColor = MaterialTheme.colors.onPrimary,
+                elevation = 0.dp
```

Comment:
Same comment about elevation and background color

Reply:
Changed the background to MaterialTheme.colors.surface, leave the elevation 0 to avoid a gray line to appear dividing the title form the tab menu, dynamically adding the elevation on scroll event not needed for this screen since the tab menu has its own divider